packages/mermaid-layout-elk/src/render.ts

```diff
@@ -3,6 +3,21 @@ import ELK from 'elkjs/lib/elk.bundled.js';
 import type { InternalHelpers, LayoutData, RenderOptions, SVG, SVGGroup } from 'mermaid';
 import { type TreeData, findCommonAncestor } from './find-common-ancestor.js';
 
+type Node = LayoutData['nodes'][number];
```

Comment:
Unrelated to this PR, but, should we rename from `Node` to something else? As `Node` conflicts with https://developer.mozilla.org/en-US/docs/Web/API/Node.

Reply:
I'm for it (if somebody actually knows a good other name for `Node`)!

To be honest, there are so many `node` parameters that don't seem to match this `Node` type that we'd probably need multiple different `NodeXXXX` types, like this `NodeWithVertex` type I made:

https://github.com/mermaid-js/mermaid/blob/6cc0132e0a192c4101c86083f34ca3af7d6ebe23/packages/mermaid-layout-elk/src/render.ts#L15-L19